doc/sphinx/changes.rst

```diff
@@ -8,6 +8,753 @@ Recent changes
+  (`#17352 <https://github.com/coq/coq/pull/17352>`_,
+  by Gaëtan Gilbert).
+- **Changed:**
+  the :tacn:`case` tactic and its variants always generate a
+  pattern-matching node, regardless of their argument. In
+  particular, they are now guaranteed to generate as many goals
+  as there are constructors in the inductive type. Previously,
+  they used to reduce to the corresponding branch when the argument
+  βι-normalized to a constructor, resulting in a single goal
+  (`#17541 <https://github.com/coq/coq/pull/17541>`_,
+  by Pierre-Marie Pédrot).
+- **Changed:**
+  :tacn:`injection` continues working using sigma types when `Eqdep_dec` has not been required even if an equality scheme was found, instead of failing
+  (`#17670 <https://github.com/coq/coq/pull/17670>`_,
+  by Gaëtan Gilbert).
+- **Removed:**
+  the no-argument form of the :tacn:`instantiate` tactic, deprecated since 8.16
+  (`#16910 <https://github.com/coq/coq/pull/16910>`_,
+  by Pierre-Marie Pédrot).
+- **Removed:**
+  undocumented tactics `hresolve_core` and `hget_evar`
+  (`#17035 <https://github.com/coq/coq/pull/17035>`_,
+  by Gaëtan Gilbert).
+- **Deprecated:**
+  the :tacn:`elimtype` and :tacn:`casetype` tactics
+  (`#16904 <https://github.com/coq/coq/pull/16904>`_,
+  by Pierre-Marie Pédrot).
+- **Deprecated:**
+  `revert dependent`, which is a misleadingly named alias of :tacn:`generalize dependent`
+  (`#17669 <https://github.com/coq/coq/pull/17669>`_,
+  by Gaëtan Gilbert).
+- **Fixed:**
+  The :tacn:`simpl` tactic now respects the :n:`simpl never` flag even
+  when the subject function is referred to through another definition.
+  (`#13448 <https://github.com/coq/coq/pull/13448>`_,
+  fixes `#13428 <https://github.com/coq/coq/issues/13428>`_,
+  by Yves Bertot).
+- **Fixed:**
+  unification is less sensitive to whether a subterm is
+  an indirection through a defined existential variable or a direct term node.
+  This results in less constant unfoldings in rare cases
+  (`#16960 <https://github.com/coq/coq/pull/16960>`_,
+  by Gaëtan Gilbert).
+- **Fixed:**
+  Fix untypable proof states generated by setoid_rewrite, which may cause some backwards-incompatibilities
+  (`#17304 <https://github.com/coq/coq/pull/17304>`_,
+  fixes `#17295 <https://github.com/coq/coq/issues/17295>`_,
+  by Lasse Blaauwbroek).
+- **Fixed:**
+  intropatterns destructing a term whose type is a product
+  cannot silently create shelved evars anymore. Instead, it
+  fails with an unsolvable variable. This can be fixed in a
+  backwards compatible way by using the e-variant of the parent
+  tactic
+  (`#17564 <https://github.com/coq/coq/pull/17564>`_,
+  by Pierre-Marie Pédrot).
+- **Changed:**
+  the unification heuristics for implicit arguments of the :tacn:`case` tactic.
+  We unconditionally recommend using :tacn:`destruct` instead, and even more so
+  in case of incompatibility.
```

Comment:
This "Changed" is misplaced.

Reply:
I didn't touch it AFAICT. Is it a bug in the script?

Reply:
I'd rather say it's a limitation. The changelog entry for #17564 contained the "Fixed" above and this "Changed", and the script does not detect and split the entries in this case. BTW, this "Changed" should have repeated the following part:

```
  (`#17564 <https://github.com/coq/coq/pull/17564>`_,
  by Pierre-Marie Pédrot).
```

Reply:
I'm not sure I understand what is wrong with the current output, TBH.

Reply:
The ordering by type of change (first "Changed", then "Removed", "Deprecated", "Added", "Fixed") ensures that users that are skimming over the changelog can quickly locate the changes that are most likely to impact compatibility.